pkg/node/publish_unpublish.go

```diff
@@ -48,7 +48,7 @@ func PublishVolume(ctx context.Context, stagingPath, containerPath string, readO
 	}
 
 	if shouldBindMount {
-		opts := []string{"bind"}
+		opts := []string{"bind", "prjquota"}
```

Comment:
What happens if there is no storage requirement specified on the requested volume?

Reply:
As we run xfs_quota on these mounts, We will add this option whenever we do a bind mount.